### PR TITLE
fix issues about template

### PR DESCRIPTION
**Pull request by kdada**
* remove `lifecycle` from initContainer
* add default topologyKey `kubernetes.io/hostname`
* fix a conflict between initContainer name with container name

### CODE REVIEW

File: templates/1.0.0/values.yaml [hunk 1]
```diff
@@ -126,14 +122,14 @@ _config:
...
-    - name: mysql
+    - name: mysql1
       type: ClusterIP
       export: true
       ports:
       - protocol: HTTP
         targetPort: 80
         port: 80
-    - name: mysql2
+    - name: mysql21
```
Comment (ddysher) [inline templates/1.0.0/values.yaml]: nit: mysql21 or mysql2?

**Comment by kdada** [comment]
@ddysher  PTAL

File: templates/1.0.0/values.yaml (hunk 1 above)
Comment (kdada) [inline templates/1.0.0/values.yaml]: Fixed

**Comment by ddysher** [comment]
LGTM

**Pull request merged**